### ISSUE TITLE
split services in enodo hub

### ISSUE THREAD

**Issue by timoj** [SiriDB/siridb-enodo-hub#76]
### TLDR;
We need to remove the scheduling part from the Hub into a seperate service. the remaining functionality of the hub will stay in the Hub service, with the difference that multiple Hub instances are possible, and will be kept up-to-date via ThingsDB

### Description
The Hub service handles API requests and incoming datapoints that the listeners send through. The Hub services holds a list with series names and their config.

### Functionality
Some agreements on functionality:
- [ ] We explicit insert series for realtime analysis. This will be a property on the series config
- [ ] Hub has no state only config
- [ ] Bulk endoints for inserting and querying
- [ ] Hub sends worker config to worker on handshake
- [ ] Hub should be able to restart without any complications. All data should go via thingdb
- [ ] The Hub uses an algorithm to determine which series goes to which worker, for easy scalability of the workers
